src/CommandResourceEstimator.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef __OPPORTUNISTIC_RESOURCE_ESTIMATOR_HPP__
+#define __OPPORTUNISTIC_RESOURCE_ESTIMATOR_HPP__
```

Comment:
You renamed the file but not the guards ;)

Reply:
thanks for noticing it! should be fixed